source/security/client-side-field-level-encryption-local-key-to-kms.txt

```diff
@@ -92,6 +93,16 @@ provider:
 
       .. include:: /includes/steps/fle-convert-to-a-remote-master-key-gcp.rst
 
+   .. tab:: KMIP KMS
+      :tabid: kmip-kms
+
+      .. important:: Java Only
```

Comment:
This is no longer true. C, C#, Go, and Python have since been released with support.
The most up-to-date status is on tickets linked to [DRIVERS-1353](https://jira.mongodb.org/browse/DRIVERS-1353).

I have been tracking in [this spreadsheet](https://docs.google.com/spreadsheets/d/1HvpsabAoSXXh3MFKT-Z5RP7cs9ZsicYb6wA6EYcRPCw/edit#gid=0) for convenience.

Reply:
Awesome, thanks for the heads up. I created tickets for adding C#, Go, and Python to this guide (I don't believe we cover C at present). Here are the tickets:

- [C#](https://jira.mongodb.org/browse/DOCSP-19924)
- [Go](https://jira.mongodb.org/browse/DOCSP-19925)
- [Python](https://jira.mongodb.org/browse/DOCSP-19926)

I remove the mentions of "Java Only" in the meantime.